app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person.get("name"), person.get("age"))
+
+    for person in people:
+        if person.get("wife"):
+            wife = Person.people[person.get("wife")]
+            Person.people[person.get("name")].wife = wife
+        elif person.get("husband"):
+            husband = Person.people[person.get("husband")]
+            Person.people[person.get("name")].husband = husband
```

Comment:
Since you already created people instances in class u could just remove one of redundant elif, by making code like this(according to dry):
``` 
   for person in people:
        if person.get("wife"):
            wife = Person.people[person.get("wife")]
            husband = Person.people[person.get("name")]
            husband.wife = wife
            wife.husband = husband
            ```